runtime/redirector/generated.c.m4

```diff
@@ -1,23 +1,12 @@
+changequote(`[',`]')dnl
 /*******************************************************************************
- * Copyright (c) 2001, 2017 IBM Corp. and others
+ * Licensed Materials - Property of IBM
+ * "Restricted Materials of IBM"
  *
- * This program and the accompanying materials are made available under
- * the terms of the Eclipse Public License 2.0 which accompanies this
- * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
- * or the Apache License, Version 2.0 which accompanies this distribution and
- * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ * (c) Copyright IBM Corp. 2001, 2017 All Rights Reserved
  *
- * This Source Code may also be made available under the following
- * Secondary Licenses when the conditions for such availability set
- * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
- * General Public License, version 2 with the GNU Classpath
- * Exception [1] and GNU General Public License, version 2 with the
- * OpenJDK Assembly Exception [2].
- *
- * [1] https://www.gnu.org/software/classpath/license.html
- * [2] http://openjdk.java.net/legal/assembly-exception.html
- *
- * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ * US Government Users Restricted Rights - Use, duplication or disclosure
```

Comment:
Looks like you reverted the copyright header to the previous version.  You will need to update it to the correct one

Reply:
This copyright header still looks wrong

Reply:
odd. i could swear i checked them. git must have botched an automated merge after a rebase. Pushing fixed versions now